HCCGo/app/css/application.less

```diff
@@ -116,9 +116,9 @@ body {
 }
 
 .card {
-  .box-shadow(0 12px 15px rgba(0, 0, 0, 0.24));
-  background-color: @whiteBackground;
-  padding: 0.5em;
+ -  .box-shadow(0 12px 15px rgba(0, 0, 0, 0.24));
+ -  background-color: @whiteBackground;
+ -  padding: 0.5em;
```

Comment:
Not entirely sure, I copied and pasted the changes from my local environment so that might be the cause